plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedList/ChangedListPresenter.java

```diff
@@ -65,21 +67,27 @@ public void onCloseClicked() {
 
     /** {@inheritDoc} */
     @Override
-    public void onCompareClicked() {
+    public void onCompareProvided() {
         comparePresenter.show(file, state, revision);
     }
 
     /** {@inheritDoc} */
     @Override
-    public void onNodeSelected(@NotNull Node node) {
+    public void onGroupByDirectoryCheckBoxValueChanged() {
+        view.setChanges(changedFiles);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void onFileNodeSelected(@NotNull Node node) {
```

Comment:
I need `onNodeNotSelected` event because when showing a dialog selection is lost and I have to send this event to tell presenter that there is no selection in view.